### ISSUE TITLE
React Native: Run iOS with custom project path

### ISSUE THREAD

**Issue by hovox** [Microsoft/vscode-react-native#477]
## Actual Behavior
`react native run-ios` is not working from vscode command palette when project's root path is not  ios folder, for my case project root is root directory of the project.

## Expected Behavior
1. calling `react native run-ios` from vscode command palette should run the app

## Software versions
- React-Native VS Code extension version: 0.3.2
- VSCode version: 1.14.0
- OS platform and version: macOS 10.12.6 Beta
- NodeJS version: 8.1.4
- React Native version: 0.42.3

Here is my `settings.json`:
```
  "javascript.validate.enable": false,
  "flow.useNPMPackagedFlow": true,
  "react-native-tools": {
    "projectRoot": ".",
    "showInternalLogs": true,
    "logLevel": "Trace"
  }
```

I think `projectRoot` config is not working, when running `React Native: Debug iOS` the config is not ignored.

**Comment by vladimir-kotikov** [comment]
@hovox, `"projectRoot"` option serves another purpose - it allows you to specify root or the _whole RN app_, not it's iOS or Android native part. This is for cases when you have RN app placed in subfolder of the project.

As a partial solution for your issue - there is [`"iosRelativeProjectPath"` option for `launch.json`](https://github.com/Microsoft/vscode-react-native/blob/master/package.json#L226) that allows you to specify root of iOS code relative to the project root.

**Comment by vladimir-kotikov** [comment]
Closing this as this has been implemented in #517
